config.go

```diff
@@ -108,17 +108,16 @@ type config struct {
 	ReadMacPath  string `long:"readonlymacaroonpath" description:"Path to write the read-only macaroon for lnd's RPC and REST services if it doesn't exist"`
 	LogDir       string `long:"logdir" description:"Directory to log output."`
 
-	Listeners   []string `long:"listen" description:"Add an interface/port to listen for connections (default all interfaces port: 9735)"`
-	ExternalIPs []string `long:"externalip" description:"Add an ip to the list of local addresses we claim to listen on to peers"`
+	RPCListeners []string `long:"rpclisten" description:"Add an interface/port to listen for RPC connections"`
+	Listeners    []string `long:"listen" description:"Add an interface/port to listen for peer connections"`
```

Comment:
While we're at it, how about adding a `--nolisten` argument? It would disable p2p listening all together, leaving only the RPC listening on `localhost`.